xml/System.Drawing/Bitmap.xml

```diff
@@ -886,13 +893,68 @@
+          <para>of the converted bitmap. This must be specified for indexed pixel formats.</para>
+          <para></para>
+          <para></para>
+          <para>This palette (called the actual palette) does not have to have the type specified by</para>
+          <para>the <paramref name="paletteType" /> parameter. The <paramref name="paletteType" /> parameter specifies a standard</para>
+          <para>palette that can be used by any of the ordered or spiral dithering algorithms. If the actual palette has a type</para>
+          <para>other than that specified by the <paramref name="paletteType" /> parameter, then</para>
+          <para>
+            <see cref="M:System.Drawing.Bitmap.ConvertFormat(System.Drawing.Imaging.PixelFormat,System.Drawing.Imaging.DitherType,System.Drawing.Imaging.PaletteType,System.Drawing.Imaging.ColorPalette,System.Single)" /> performs a nearest-color</para>
+          <para>conversion from the standard palette to the actual palette.</para>
+          <para></para>
+        </param>
+        <param name="alphaThresholdPercent">
+          <para></para>
+          <para>Real number in the range 0 through 100 that specifies which pixels in the source bitmap will map to the</para>
+          <para>transparent color in the converted bitmap.</para>
+          <para></para>
+          <para></para>
+          <para>A value of 0 specifies that none of the source pixels map to the transparent color. A value of 100</para>
+          <para>specifies that any pixel that is not fully opaque will map to the transparent color. A value of t specifies</para>
+          <para>that any source pixel less than t percent of fully opaque will map to the transparent color. Note that for</para>
+          <para>the alpha threshold to be effective, the palette must have a transparent color. If the palette does not have</para>
+          <para>a transparent color, pixels with alpha values below the threshold will map to color that most closely</para>
+          <para>matches (0, 0, 0, 0), usually black.</para>
+          <para></para>
+        </param>
+        <summary>
+          <para>Converts the bitmap to the specified <paramref name="format" /> using the given <paramref name="ditherType" />.</para>
+          <para>The original pixel data is replaced with the new format.</para>
+        </summary>
+        <remarks>
+          <para></para>
+          <para>
+            <paramref name="paletteType" /> and <paramref name="palette" /> really only have relevance with indexed pixel</para>
+          <para>formats. You can pass a <see cref="T:System.Drawing.Imaging.ColorPalette" /> for non-indexed pixel formats, but it has no impact on the</para>
+          <para>transformation and will effective just call <see cref="P:System.Drawing.Image.Palette" /> to set the palette when the conversion</para>
+          <para>is complete.</para>
+          <para></para>
+        </remarks>
```

Comment:
The triple slash comments had unnecessary newlines so they ended up getting ported with empty `<para>` items here. I'll have to fix it in the tool but meanwhile I'll provide some suggested changes. A lot of information needs to be moved to remarks:

```suggestion
        <param name="format">The new pixel format.</param>
        <param name="ditherType">The dithering algorithm.</param>
        <param name="paletteType">The palette type to use when the pixel format is indexed.</param>
        <param name="palette">Pointer to a <see cref="T:System.Drawing.Imaging.ColorPalette" /> that specifies the palette whose indexes are stored in the pixel data of the converted bitmap. This must be specified for indexed pixel formats.</param>
        <param name="alphaThresholdPercent">A real number in the range 0 through 100 that specifies which pixels in the source bitmap will map to the transparent color in the converted bitmap.</param>
        <summary>Converts the bitmap to the specified <paramref name="format" /> using the given <paramref name="ditherType" />. The original pixel data is replaced with the new format.</summary>
        <remarks>
          <para>For <paramref name="format" />, <see cref="F:System.Drawing.Imaging.PixelFormat.Format16bppGrayScale" /> is not supported.</para>
          <para>For <paramref name="ditherType" />, pass <see cref="F:System.Drawing.Imaging.DitherType.None" /> when the conversion does not reduce the bit depth of the pixel data. This must be <see cref="F:System.Drawing.Imaging.DitherType.Solid" /> or <see cref="F:System.Drawing.Imaging.DitherType.ErrorDiffusion" /> if the <paramref name="paletteType" /> is <see cref="F:System.Drawing.Imaging.PaletteType.Custom" /> or <see cref="F:System.Drawing.Imaging.PaletteType.FixedBlackAndWhite" />.</para>
          <para>
            <paramref name="paletteType" /> is ignored for non-indexed pixel formats.
          </para>
          <para>The <paramref name="palette" /> (called the actual palette) does not have to have the type specified by the <paramref name="paletteType" /> parameter. The <paramref name="paletteType" /> parameter specifies a standard palette that can be used by any of the ordered or spiral dithering algorithms. If the actual palette has a type other than that specified by the <paramref name="paletteType" /> parameter, then <see cref="M:System.Drawing.Bitmap.ConvertFormat(System.Drawing.Imaging.PixelFormat,System.Drawing.Imaging.DitherType,System.Drawing.Imaging.PaletteType,System.Drawing.Imaging.ColorPalette,System.Single)" /> performs a nearest-color conversion from the standard palette to the actual palette.</para>
          <para>For <paramref name="alphaThresholdPercent">, a value of 0 specifies that none of the source pixels map to the transparent color. A value of 100 specifies that any pixel that is not fully opaque will map to the transparent color. A value of t specifies that any source pixel less than t percent of fully opaque will map to the transparent color. Note that for the alpha threshold to be effective, the palette must have a transparent color. If the palette does not have a transparent color, pixels with alpha values below the threshold will map to color that most closely matches <c>(0, 0, 0, 0)</c>, usually black.</para>
          <para>
            <paramref name="paletteType" /> and <paramref name="palette" /> really only have relevance with indexed pixel formats. You can pass a <see cref="T:System.Drawing.Imaging.ColorPalette" /> for non-indexed pixel formats, but it has no impact on the transformation and will effective just call <see cref="P:System.Drawing.Image.Palette" /> to set the palette when the conversion is complete.
          </para>
        </remarks>
```